zeppelin/executors/notebook_executor.py

```diff
@@ -29,10 +29,17 @@ def wait_for_notebook_to_execute(self):
         while True:
             r = requests.get('http://{0}/api/notebook/job/{1}'.format(
                              self.zeppelin_url, self.notebook_id))
+
+            if r.status_code != 200:
+                print('Internal Server Error (500). Trying again in 5 seconds...')
```

Comment:
I looked at the Zeppelin API. I find it odd that they return a 500 for the status of the paragraphs not being completed yet. Can we print something less scary? Something like, "Notebook still running, checking again in 5 seconds..."?